### PR TITLE
Feature desktop

### PR DESCRIPTION
**Pull request by Obote**
### What does this PR do?
- Create UIs adaptable to different screen sizes
- Use media queries.
- Use Figma to create UI
- Configure linters
- Use GitHub flow
- Documentation

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -11,6 +11,14 @@
...
         <header class="navigation">
             <a class="logo" href="#">Obote</a>
+            <div class="menu-items">
+                <ul>
+                    <li><a href="#">Profile</a></li>
+                    <li><a href="#">About</a></li>
+                    <li><a href="#">Contact</a></li>
+                    <li><a href="#"> <img src="./assets/icon/Union.svg" alt="message icon"></a></li>
+                </ul>
+            </div>
```
Comment (Emmyn5600) [inline index.html:21]: - [ ] Remove the white space that is appearing at the top of the menus
header5

File: index.html [hunk 2]
```diff
@@ -139,11 +205,29 @@ <h3>CONTACT US</h3>
...
         </section>
+
+        <section class="form-container" id="desktop-form">
+            <h3>Let’s chat!</h3>
+            <P>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </P>
+            <form action="https://formspree.io/f/xdoylybq" method="POST">
+                <div class="form-box">
+                    <label for="name"></label>
+                    <input class="input1" type="text" id="firstName" name="name" placeholder="Text Input" required>
+                    <label for="name"></label>
```
Comment (Emmyn5600) [inline index.html]: - [ ] Kindly, rename the placeholder `Text Input` with `First Name`
firstname

File: index.html [hunk 3]
```diff
@@ -156,5 +240,26 @@ <h3>CONTACT US</h3>
...
+                <li class="footer-disposal"><a href="#"><img src="./assets/icon/medium.png" alt="Medium"></a></li>
+            </ul>
+            </div>
+            <div class="bottom">
+                <a href="#" class="footer-email">obotedenis256@gmail.com</a>
+                <img src="./assets/images/footer-shape.png" alt="footer line">
+            </div>  
+        </div>
+    </footer>
+
```
Comment (Emmyn5600) [inline index.html:263]: - [ ] Please fix the overflow-x  that is appearing at the end of your page. Make it as `overflow-x:hidden`
footer3

**Review by Emmyn5600 — CHANGES_REQUESTED** [review]
## STATUS: Changes Requested :recycle:

Hi @Obote 

Good job so far!:clap::clap:
You have done well in implementing the project requirements :clap:, However, there are some issues that you still need to work on to go to the next project but you are almost there!:muscle:

##  Required Changes:writing_hand:
  
**_Check the comments under the review._**

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

#### _**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_
------------------------------------------------------------------------------------------------------------------------------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

File: index.html (hunk 3 above)
Comment (Obote) [reply index.html:263]: Thank you so much for the time and effort you have put to review my project, the suggestions are very much welcomed. :mechanical_arm:

**Review by Shaher-11 — APPROVED** [review]
#### Hello @Obote,

# Congratulations! 🎉 
#### Your project is approved!  There is nothing else to say other than... it's time to merge it :muscle: :muscle::muscle:

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Cheers and Happy coding!👏👏👏

**Pull request merged**